### ISSUE TITLE
Watch the development (and a bit of an update)

### ISSUE THREAD

**Issue by chabad360** [chabad360/covey#10]
I've started to occasionally live-stream the development on my [Youtube channel](https://youtube.com/c/MendelG), if you want a bit more of a glimpse into the development of Covey.

The development has slowed primarily due two things:
1. Work has picked up a lot, so I haven't been able to spend as much time focusing on Covey.
2. I've been waiting on traefik/yaegi#919 so that I can finish the implementation of the [plugin system](/chabad360/plugins) and then I can release the alpha.

In the meantime I've been working on CI portions

**Comment by chabad360** [comment]
After encountering another bug with Yaegi, I put development on hold for a bit while I worked on some other projects. But after having a bit of an epiphany and doing further research, it appears that I've manged to work around the issue. In which case, I'll hopefully have an alpha before the end of January (provided nothing else work related gets in the way, which it might...).

**Comment by chabad360** [comment]
Another update: Due to a lack of motivation, I'm putting this project on hold for about a month or two, while I work on creating the actual cluster this project was meant to help with managing. Being able to dogfood this project will help make the development process happen a lot faster, but getting there will take me a little while. So hold tight.
